packages/Popover/src/Popover.js

```diff
@@ -175,6 +180,8 @@ class Popover extends React.Component {
     // about setState for Popovers and Tooltips in ComponentDidMount
     // https://reactjs.org/docs/react-component.html#componentdidmount
 
+    focusableElements = document.querySelectorAll(focusableElementSelector);
```

Comment:
I think this should be set when the Popover opens.  The elements on the page could vary while the Popover is mounted.